http-ballerina-tests/tests/Config.toml

```
@@ -0,0 +1,5 @@
+[http_tests]
+backendEP_port=7070
```

Shall we update as follows? @chamil321 
```suggestion
backendPort=7070
```

---

+1